packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -567,6 +599,36 @@ export class TextFieldBase extends React.Component<ITextFieldProps, ITextFieldSt
       textField.style.height = textField.scrollHeight + 'px';
     }
   }
+
+  private _shouldShowRevealButton = (): boolean => {
+    if (typeof window === 'undefined' || !window.navigator || !window.navigator.userAgent) {
+      return false;
+    }
+
+    if (this.props.canRevealPassword) {
+      const userAgent = window.navigator.userAgent;
```

Comment:
hmm. should this be an exception list where this is functional by default?

Reply:
I'm having a hard time understanding why we should conditionally only allow the behavior on a specific list and why we wouldn't allow it otherwise?

Reply:
If a new browser comes along that isn't handled, I was thinking it would be a better experience for the end user if the reveal password button was not shown than to potentially have a browser's built in reveal password and this additional reveal password button.

Basically, I'm looking for the least harm way this would fail.

Reply:
Switched this to only not show the reveal button on IE and Edge